content/docs/reference-events.md

```diff
@@ -60,15 +60,15 @@ function onClick(event) {
 }
 ```
 
-> Note:
+> ध्यान दें:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
+> यदि आप एक asynchronous तरीके से इवेंट के प्रॉपर्टीज का उपयोग करना चाहते हैं, तो आपको इवेंट पर `event.persist()` को कॉल करना चाहिए, जो पूल से `सिंथेटिक इवेंट` को हटा देगा और उपयोगकर्ता कोड द्वारा इवेंट के संदर्भ को बनाए रखने की अनुमति देगा।
```

Comment:
`सिंथेटिक इवेंट` -> remove quotes

Reply:
और उपयोगकर्ता कोड द्वारा इवेंट के संदर्भ को बनाए रखने की अनुमति देगा। gives the sentence a different meaning than original

Reply:
संदर्भ  can be references in hindi since it is a technical term